README.md

```diff
@@ -126,6 +126,15 @@ a := [3]byte{
 }
 ```
 
+#### python array: -ap
+
+```sh
+$ hx -ap -c8 tests/files/tiny.txt
+a = [
+    0x69, 0x6c, 0x0a
+]
+```
+
 ## manual
 
 ```txt
```

Comment:
@jmg-duarte thank you for you contribution. could you also update the manual section `Set source code format output` to include the Python array output?

Reply:
Will take care of that, also if you're interested I can add more languages, such as Kotlin and Java

Reply:
great, was just looking at adding Kotlin recently. feel free to add support for Kotlin and Java.